site/docs-md/apis/camera/index.md

```diff
@@ -88,8 +89,8 @@ export class CameraPage {
       source: CameraSource.Camera
     });
     
-    // Example of using the Base64 return type. It's recommended to use CameraResultType.Uri
-    // instead for performance reasons when showing large, or a large amount of images.
+    // Recommended to use CameraResultType.DataUrl instead of Base64 for 
```

Comment:
DataUrl is as bad as Base64, recommended is Uri

Reply:
Updated